lib/elixir/test/elixir/string/normalizer_test.exs

```diff
@@ -0,0 +1,49 @@
+Code.require_file "../test_helper.exs", __DIR__
+
+defmodule String.NormalizerTest do
+  use ExUnit.Case, async: true
+
+  # String.normalize
+
+  test "String.normalize 015C;LATIN CAPITAL LETTER S WITH CIRCUMFLEX" do
```

Comment:
Maybe better to group all the `String.normalize` tests under one `test` like:
```elixir
test "String.normalize/2" do
  # 015C;LATIN CAPITAL LETTER S WITH CIRCUMFLEX
  assert String.normalize("\u015C", :nfd) == "\u0053\u0302"
  assert String.normalize("\u0053\u0302", :nfc) == "\u015C"
  # 015D;LATIN SMALL LETTER S WITH CIRCUMFLEX
  assert String.normalize("\u015D", :nfd) == "\u0073\u0302"
  assert String.normalize("\u0073\u0302", :nfc) == "\u015D"
  # ...
end
```
?